RetroAchievements/RAWeb
# refactor(reportissue): migrate from Knockout.js to Alpine

This PR migrates _reportissue.php_'s client-side code from Knockout.js to Alpine. Once this and https://github.com/RetroAchievements/RAWeb/pull/1876 are merged, there will be nothing left in the project depending upon Knockout.js and it can be removed as a dependency.

This is a 1:1 conversion with no feature/function changes.

**How to test this PR**
Visit localhost/reportissue.php?i=116522.

* In the Description field, try typing the string "manual unlock". You should see some helper text appear.
* In the Description field, try typing the string "didn't work". Helper text should appear and the submit button should be disabled.
* In the Emulator field, select "Bizhawk". The "Core" field should vanish.
